# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1523271876 14400
#      Mon Apr 09 07:04:36 2018 -0400
# Branch stable
# Node ID c38c70a5222cc4093ad63a420bd85db315d5deb5
# Parent  8c38361718a973e96c49a0bc3cec4f80e2b95cd4
* repelem.m: Update copyright year.

diff --git a/scripts/general/repelem.m b/scripts/general/repelem.m
--- a/scripts/general/repelem.m
+++ b/scripts/general/repelem.m
@@ -1,9 +1,9 @@
-## Copyright (C) 2015 Markus Bergholz
+## Copyright (C) 2015-2018 Markus Bergholz
 ##
 ## This file is part of Octave.
 ##
 ## Octave is free software: you can redistribute it and/or modify it
 ## under the terms of the GNU General Public License as published by
 ## the Free Software Foundation, either version 3 of the License, or
 ## (at your option) any later version.
 ##
